crates/trie/common/src/proofs.rs

```diff
@@ -18,17 +18,17 @@ use derive_more::{Deref, DerefMut, IntoIterator};
 use itertools::Itertools;
 use reth_primitives_traits::Account;
 
-/// Proof targets map.
+/// Map representing proof targets that we have already fetched.
 #[derive(Deref, DerefMut, IntoIterator, Clone, PartialEq, Eq, Default, Debug)]
-pub struct MultiProofTargets(B256Map<B256Set>);
+pub struct FetchedProofTargets(B256Map<B256Set>);
 
-impl FromIterator<(B256, B256Set)> for MultiProofTargets {
-    fn from_iter<T: IntoIterator<Item = (B256, B256Set)>>(iter: T) -> Self {
+impl FromIterator<(B256, B256Set)> for FetchedProofTargets {
+    fn from_iter<I: IntoIterator<Item = (B256, B256Set)>>(iter: I) -> Self {
         Self(B256Map::from_iter(iter))
     }
 }
 
-impl MultiProofTargets {
+impl FetchedProofTargets {
     /// Creates an empty `MultiProofTargets` with at least the specified capacity.
```

Comment:
comments here and in other methods need an update for the new struct name, and also let's refer to them as an item
```suggestion
    /// Creates an empty [`FetchedProofTargets`] with at least the specified capacity.
```